jupyterhub/Chart.yaml

```diff
@@ -1,5 +1,5 @@
 name: jupyterhub
-version: v0.7-dev
+version: v0.7-2940c96a
```

Comment:
Is this a side effect of running `chartpress` and should be undo? Not sure.

Reply:
Yepp it is! I included the changes by mistake to a commit!